framework/lib/components/menu/menu.tsx

```diff
@@ -1,24 +1,102 @@
 import React from 'react'
-import { Menu as ChakraMenu, MenuProps } from '@chakra-ui/react'
+import { Menu as ChakraMenu } from '@chakra-ui/react'
+import { MenuProps } from './types'
 
 /**
  * @see {@link https://northlight.dev/reference/menu}
  *
- * @example
+ * @example (Example)
+ * ##Clean menu with different options
+ * The menu can be `relaxed` for primary menus, and `compact` for secodnary menus.
+ * <br />
  * (?
- *<Menu>
-  <MenuButton as={Button} rightIcon={<ChevronDownDuo />}>
-    Actions
-  </MenuButton>
-  <MenuList>
-    <MenuItem>Download</MenuItem>
-    <MenuItem>Create a Copy</MenuItem>
-    <MenuItem>Mark as Draft</MenuItem>
-    <MenuItem>Delete</MenuItem>
-    <MenuItem>Attend a Workshop</MenuItem>
-  </MenuList>
-</Menu>
- * ?)
+ * +
+ * const MyMenuComponent = () => {
+ * const [selectedLabel, setSelectedLabel] = useState('Compare periods');
  *
- */
-export const Menu = (props: MenuProps) => <ChakraMenu { ...props } />
+ *   const menuOptions = {
+ *     '1': 'No Comparison',
+ *     '3': 'Previous week',
```

Comment:
is there no '2'?

Reply:
Not a big fan of number 2 haha. I forgot to put value="2" as well. Just need to update the numbers. Thnk